src/components/showcase/Showcase.grid.tsx

```diff
@@ -18,6 +18,96 @@ export let getCollection: any;
 export interface ShowcaseGridInterface {
   collection: string;
 }
+// export interface NFTCardInterface {
+//     asset:
+// }
+
+const NFTCard = ({ asset = {}, id = 0}) => {
+    function truncateString(str = '', n = 0) {
+        if (str.length > n) {
+            return str.substring(0, n) + "...";
+        } else {
+            return str;
+        }
+    }
+
+    return (
+        <Link
+            key={id}
+            href={`/details/${id}`}
+            transition="opacity 0.3s .3s ease-in"
+            display="flex"
+            flexFlow="column nowrap"
+            backgroundColor="rgba(255,255,255,0.8)"
+            backdropFilter="blur(3px)"
+            borderRadius="6px"
+            overflow="hidden"
+            boxShadow="0 0 10px rgba(0,0,0,0.7)"
+            position="relative"
+            sx={{
+                "&::after": {
+                    content: "''",
+                    backgroundImage: "url(/assets/ntfa-logo.png)",
+                    backgroundSize: "100%",
+                    backgroundRepeat: "no-repeat",
+                    filter: "drop-shadow(0 0 5px rgba(0,0,0,.4))",
+                    position: "absolute",
+                    bottom: 0,
+                    right: 0,
+                    height: "60px",
+                    width: "60px",
+                    transform: "translate3d(0, 40px, 0)",
+                    opacity: 0,
+                    transition: "transform 2.4s 0.4s ease-in-out, opacity 0.2s 0.4s ease"
+                },
+                "&:hover": {
+                    boxShadow: `0 0 5px rgba(0,0,0,0.4)`,
+                "&::after": {
+                    transform: "translate3d(0, -400%, 0)",
+                    opacity: 1
+                    },
+                "&::before": {
+                    transform: "translate3d(0, 10px, 0)",
+                    opacity: 1
+                }
+                },
+            }}
+        >
+
+            <Box
+            className="playerWrapper"
+            position="relative"
+            paddingTop={`${(212 / 373) * 100}%`}
+            height="0"
+            width="100%"
+            maxWidth="512px"
+            zIndex={200}
+            overflow="hidden"
+            >
+                <ReactPlayer
+                    url={asset.path}
```

Comment:
@c0mput3rxz here, ts is complaining that path does not exist on type `asset` and I know why it's complaining but how do i set up an interface to handle what is in the `asset` object?

Reply:
Ignore that. I googled and struck gold.